### ISSUE TITLE
Any plan to support amqp1-0-0?

### ISSUE THREAD

**Issue by singlaive** [python-trio/trio-amqp#7]
I understand amqp1-0-0 is quite different with 0-9-1, but since the former is the formal release, are you considering supporting it?

**Comment by njsmith** [comment]
Disclaimer: I'm not an expert on AMQP or this library, but @smurfix has been pretty busy lately, so in case he doesn't see this I'll give you my best guess :-).

I don't think anyone is currently working on supporting AMQP 1-0-0 in this library. Also, my understanding is that AMQP 0-9-1 and AMQP 1-0-0 are entirely unrelated protocols, and from a quick google AFAICT they're usually handled by different client libraries (i.e. I see AMQP 0-9-1 client libraries, and AMQP 1-0-0 client libraries, not libraries that do both).

That said, it would certainly be awesome to have an AMQP 1-0-0 library for trio! And it's totally possible. Just I think you'd probably be better off starting from some other AMQP 1-0-0 library, or from scratch, rather than expecting this library to solve it.

Hope that helps!

**Comment by singlaive** [comment]
I agree with your feeling. I could try implement amqp1-0 for trio but at the moment I am looking for some handy out-of-box library on production after I chose trio in our service.
Hmmm..shame
